3-enrich/stream-enrich/kinesis/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/KinesisEnrich.scala

```diff
@@ -165,10 +164,10 @@ object KinesisEnrich extends Enrich {
         val file = new File(filepath)
         if (file.exists) Source.fromFile(file).mkString.success
         else "Iglu resolver configuration file \"%s\" does not exist".format(filepath).failure
-      case DynamoDBRegex(region, table, key) =>
+      case DynamoDBRegex(endpoint, table, key) => //double check
```

Comment:
@Saud-Naumani please research the impact of this change

Reply:
I ran the debugger with a breakpoint at this line and it doesn't even get hit. I also reverted this entire file to what we have in master and it had zero effect; everything still works fine.